src/lxc/lxccontainer.h

```diff
@@ -66,11 +66,11 @@ enum {
 };
 
 /* Container's specific file names */
-#define LXC_CONFIG_FNAME       "config"
-#define LXC_PARTIAL_FNAME      "partial"
-#define LXC_ROOTFS_DNAME       "rootfs"
-#define LXC_TIMESTAMP_FNAME    "ts"
-#define LXC_COMMENT_FNAME      "comment"
+#define LXC_CONFIG_FNAME      "config"
+#define LXC_PARTIAL_FNAME     "partial"
+#define LXC_ROOTFS_DNAME      "rootfs"
+#define LXC_TIMESTAMP_FNAME   "ts"
+#define LXC_COMMENT_FNAME     "comment"
```

Comment:
This should not be a separate commit.

Reply:
If I can't modify lxccontainer.h, should I define the macro names at the top level of lxccontainer.c ?

Reply:
Or you put them in `macro.h`.